t/cobrand/tfl.t

```diff
@@ -417,12 +412,14 @@ subtest "extra information included in email" => sub {
     $mech->clear_emails_ok;
     FixMyStreet::Script::Reports::send();
     my @email = $mech->get_email;
-    like $email[0]->header('From'), qr/"Joe Bloggs" <fms-report-$id-/;
+    is $email[1]->header('From'), '"TfL Streetcare" <fms-tfl-DO-NOT-REPLY@example.com>';
```

Comment:
This is now same check as line 424, should it still be checking 0, not 1?

Reply:
Oops! Ta.